src/iuse.cpp

```diff
@@ -6411,20 +6487,769 @@ struct npc_photo_def : public JsonDeserializer, public JsonSerializer {
+            if( guy ) {
+                if( guy->get_movement_mode() == "crouch" ) {
+                    pose = _( "sits" );
+                } else {
+                    pose = _( "stands" );
+                }
+                auto vec = guy->short_description_parts();
+                figure_appearance = join( vec, "\n\n" );
+                figure_name = guy->name;
+                pronoun_sex = guy->male ? _( "He" ) : _( "She" );
+                creature = guy;
+                player_vec.push_back( guy );
+            } else {
+                if( mon->is_hallucination() || mon->type->in_species( HALLUCINATION ) ) {
+                    continue; // do not include hallucinations
+                }
+                pose = _( "stands" );
+                figure_appearance = "\"" + mon->type->get_description() + "\"";
+                figure_name = mon->name();
+                pronoun_sex = pgettext( "Pronoun", "It" );
+                creature = mon;
+                monster_vec.push_back( mon );
+            }
+
+            figure_effects = effects_description_for_creature( creature, pose, pronoun_sex );
+            description_figures_appearance[ figure_name ] = figure_appearance;
+
+            object_names_collection obj_collection = enumerate_objects_around_point( current, 1, aim_point, 2,
+                    camera_pos, min_visible_volume, true,
+                    ignored_points, vehicles_recorded );
+            std::string figure_text = pose + obj_collection.figure_text;
+
+            if( !figure_effects.empty() ) {
+                figure_text += " " + figure_effects;
+            }
+            if( !obj_collection.obj_nearby_text.empty() ) {
+                figure_text += " " + obj_collection.obj_nearby_text;
+            }
+            auto name_text_pair = std::pair<std::string, std::string>( figure_name, figure_text );
+            if( current == aim_point ) {
+                description_figures_status.insert( description_figures_status.begin(), name_text_pair );
+            } else {
+                description_figures_status.push_back( name_text_pair );
+            }
+        }
+    }
+
+    // scan for everythin NOT near critters
+    object_names_collection obj_coll = enumerate_objects_around_point( aim_point, 2, aim_point, 2,
+                                       camera_pos, min_visible_volume, false,
+                                       ignored_points, vehicles_recorded );
+
+    std::string photo_text = _( "This is a photo of " );
+
+    bool found_item_aim_point;
+    std::string furn_desc = feature_description_at_point( aim_point, found_item_aim_point, 0_ml ) ;
+    item item = get_top_item_at_point( aim_point, 0_ml );
+    std::string trap_name = trap_name_at( aim_point );
+    std::string ter_name = colorized_ter_name_flags_at( aim_point, {}, {} );
+    std::string field_desc = get_field_description_at( aim_point );
+
+    bool  found_vehicle_aim_point = g->m.veh_at( aim_point ).has_value(),
```

Comment:
surprised astyle didn't catch it but looks like there's 2 spaces here
```suggestion
    bool found_vehicle_aim_point = g->m.veh_at( aim_point ).has_value(),
```